stl/inc/yvals_core.h

```diff
@@ -684,6 +684,10 @@
 #define _HAS_UNEXPECTED (!_HAS_CXX17)
 #endif // _HAS_UNEXPECTED
 
+#if _HAS_UNEXPECTED && _HAS_CXX23
+#error STL1004: C++98 unexpected() is incompatible with C++23 unexpected<E>.

```

Comment:
I find it hilariously ironic that the other occurrences of `#error STLXXXX` are all `#error STLXXXX: Unexpected...` whereas this one is actually about the identifier `unexpected`. (No change requested. Unless you want to change this to `STL1004: Unexpected unexpected().` which would be a terrible but amusing error message.)